x-pack/plugins/observability/server/saved_objects/slo.ts

```diff
@@ -38,6 +37,21 @@ export const slo: SavedObjectsType = {
   name: SO_SLO_TYPE,
   hidden: false,
   namespaceType: 'multiple-isolated',
+  switchToModelVersionAt: '8.10.0',
```

Comment:
💬 I tried setting this to 8.12.0 but the framework was complaining.

Reply:
You effectively don't need to set this option, as it gets applied with the default/max value (`8.10.0`) if unspecified . It was exposed for type owners that wanted to switch before `8.10.0` (and for testing purposes)